apisix/plugins/openid-connect.lua

```diff
@@ -40,7 +41,11 @@ local schema = {
         logout_path = {type = "string"}, -- default is /logout
         redirect_uri = {type = "string"}, -- default is ngx.var.request_uri
         public_key = {type = "string"},
-        token_signing_alg_values_expected = {type = "string"}
+        token_signing_alg_values_expected = {type = "string"},
+        set_access_token_header = {type = "boolean"}, -- default is true
```

Comment:
Let's use the new way to set the default value:
https://github.com/apache/apisix/blob/9dfe697b644691509b55cd7ec58188b33dd3e792/apisix/plugins/cors.lua#L37

Reply:
Sure, hadn't seen it before.

Reply:
Sure, hadn't seen it before. Will change.

Reply:
I've changed the scheme for the options that I've added. But have left the existing options untouched to not make this PR any larger.